ArjunaJTA/narayana-jta-jakarta/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <parent>
+    <groupId>org.jboss.narayana.jta</groupId>
+    <artifactId>narayana-jta-all</artifactId>
+    <version>5.11.2.Final-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>narayana-jta-jakarta</artifactId>
+  <name>Narayana: ArjunaJTA Jakarta narayana-jta-jakarta</name>
+  <description>Narayana: ArjunaJTA Jakarta narayana-jta-jakarta (jta uber jar)</description>
+  <packaging>jar</packaging>
+
+  <properties>
+    <version.jakarta.transaction.jakarta.transaction-api>2.0.0</version.jakarta.transaction.jakarta.transaction-api>
+    <version.jakarta.resource.jakarta.resource-api>2.0.0</version.jakarta.resource.jakarta.resource-api>
+    <version.org.jboss.jboss-transaction-spi>7.6.1.Final</version.org.jboss.jboss-transaction-spi>
```

Comment:
To me the PR is good for merge. Just I would like to ask for one more adjustment, please mov the version configuration to root `pom.xml`. I mean redefine the version 7.6.0 here: https://github.com/jbosstm/narayana/blob/master/pom.xml#L485
and remove the line from this pom.xml.

Reply:
Done